backend/code_review_backend/issues/tests/test_diff.py

```diff
@@ -40,10 +41,17 @@ def setUp(self):
                 mercurial_hash=hashlib.sha1(f"hg {i}".encode("utf-8")).hexdigest(),
             )
 
+        # Force created date update without using inner django trigger
+        # so that all diffs in the test have the same date to be able
+        # to compare the payload easily
+        self.now = datetime.utcnow().isoformat() + "Z"
+        Diff.objects.update(created=self.now)
+
     def test_list_diffs(self):
         """
         Check we can list all diffs with their revision
         """
+        self.maxDiff = None
```

Comment:
You're not using this

Reply:
It's a [debug trigger for the unit tests](https://docs.python.org/3.4/library/unittest.html#unittest.TestCase.maxDiff)... forgot to remove it